### PR TITLE
docs(qa): Update instructions on the QA methodology

### PR DESCRIPTION
**Pull request by hvanz**
[Rendered](https://github.com/cometbft/cometbft/blob/hvanz/qa-update-instructions/docs/references/qa/method.md)

The section on Extracting Results depends on new scripts added by https://github.com/cometbft/qa-infra/pull/61.

---

#### PR checklist

- [ ] Tests written/updated
- [ ] Changelog entry added in `.changelog` (we use [unclog](https://github.com/informalsystems/unclog) to manage our changelog)
- [ ] Updated relevant documentation (`docs/` or `spec/`) and code comments
- [X] Title follows the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) spec

### CODE REVIEW

**Review by melekes — APPROVED** [review]

File: docs/references/qa/method.md [hunk 1]
```diff
@@ -47,54 +64,71 @@ _200 Node Test_, and _Rotating Nodes Test_.
 
 ## 200 Node Testnet
 
+This test consists in spinning up 200 nodes (175 validators + 20 full nodes + 5 seed nodes) and
+performing two of experiments:
```
Comment (sergio-mena) [inline docs/references/qa/method.md:68]:
```suggestion
performing two experiments:
```

File: docs/references/qa/method.md [hunk 2]
```diff
@@ -47,54 +64,71 @@ _200 Node Test_, and _Rotating Nodes Test_.
...
+- First we find the [saturation point](saturation) of the network by running the script
+  [200-node-loadscript.sh][200-node-loadscript.sh].
+- Then we run several times the testnet using the saturation point to collect data.
+
+The script [200-node-loadscript.sh] runs multiple transaction load instances with all possible
+combinations of the following parameters:
+- number of transactions sent per second (the rate): 200, 400, 800, and 1600.
+- number of connections to the target node: 1, 2, and 4.
```
Comment (sergio-mena) [inline docs/references/qa/method.md:76]: We probably can drop 2 and 4 here, and just use 1 connections, as we never saw any impact in this test case when changing the number of connections. @lasarojc what do you think?

**Review by sergio-mena — APPROVED** [review]
🙏

File: docs/references/qa/method.md (hunk 2 above)
Comment (hvanz) [reply docs/references/qa/method.md:76]: Totally agree. Most instances with 400 tx/s and above are anyway saturated. We could consider refining the rate values, something like from 200 to 1000, in intervals of 100.

**Review by melekes — APPROVED** [review]
👍

**Pull request merged**